README.md

```diff
@@ -140,3 +142,40 @@ This allows to delete a specific reminder.
 
 * 204 if request succeeded.
 * 404 if no reminder with this ID exists.
+
+## API notifications
+
+### POST `/notifications`
```

Comment:
Perhaps we could use the name `/subscription` since you are managing the push notifications subscription.

Here is creating, but there are other steps like resubscribing, so having a named end point just for managing subscriptions (or even unsubscribe) sounds better to me.